app/controllers/beers_controller.rb

```diff
@@ -0,0 +1,41 @@
+class BeersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_beer, only: [:show, :edit, :update, :destroy]
+
+  def index
+    if params[:search]
+      @beers = Beer.where('name LIKE ?', "%#{params[:search]}%")
```

Comment:
This can be extracted to a `private` method:

```
def search
  Beer.where('name LIKE ?', "%#{params[:search]}%")
end
```

then you can Use a ternary on this index method like this:

```
@beers = if params[:search] ? search : Beer.all
```

so you can declare once `@beers`

of if you don't like the approach you can do the following:

```
@beers = if params[:search]
           Beer.where('name LIKE ?', "%#{params[:search]}%")
         else
           Beer.all
         end
```

What do you think?